hack/dashboard/assets/dashboard.json

```diff
@@ -1,967 +1,798 @@
-				"useTags": false
-			},
-			{
-				"allValue": ".*",
-				"current": {
-					"selected": false,
-					"text": "All",
-					"value": "$__all"
-				},
-				"datasource": {
-					"type": "prometheus",
-					"uid": "${datasource}"
-				},
-				"definition": "label_values(kepler_container_package_joules_total{container_namespace=~\"$namespace\"}, pod_name)",
-				"hide": 0,
-				"includeAll": true,
-				"label": "Pod",
-				"multi": false,
-				"name": "pod",
-				"options": [],
-				"query": {
-					"query": "label_values(kepler_container_package_joules_total{container_namespace=~\"$namespace\"}, pod_name)",
-					"refId": "StandardVariableQuery"
-				},
-				"refresh": 1,
-				"regex": "",
-				"skipUrlSync": false,
-				"sort": 0,
-				"tagValuesQuery": "",
-				"tagsQuery": "",
-				"type": "query",
-				"useTags": false
-			},
-			{
-				"current": {
-					"selected": false,
-					"text": "2.23",
-					"value": "2.23"
-				},
-				"hide": 0,
-				"label": "Coal Coeff",
-				"name": "coal",
-				"options": [
-					{
-						"selected": true,
-						"text": "2.23",
-						"value": "2.23"
-					}
-				],
-				"query": "2.23",
-				"skipUrlSync": false,
-				"type": "textbox"
-			},
-			{
-				"current": {
-					"selected": false,
-					"text": "0.91",
-					"value": "0.91"
-				},
-				"hide": 0,
-				"label": "Natural Gas Coeff",
-				"name": "natural_gas",
-				"options": [
-					{
-						"selected": true,
-						"text": "0.91",
-						"value": "0.91"
-					}
-				],
-				"query": "0.91",
-				"skipUrlSync": false,
-				"type": "textbox"
-			},
-			{
-				"current": {
-					"selected": false,
-					"text": "2.13",
-					"value": "2.13"
-				},
-				"hide": 0,
-				"label": "Petroleum Coeff",
-				"name": "petroleum",
-				"options": [
-					{
-						"selected": true,
-						"text": "2.13",
-						"value": "2.13"
-					}
-				],
-				"query": "2.13",
-				"skipUrlSync": false,
-				"type": "textbox"
-			},
-			{
-				"description": "1W*s = 1J and 1J = (1/3600000)kWh",
-				"hide": 2,
-				"label": "",
-				"name": "watt_per_second_to_kWh",
-				"query": "0.000000277777777777778",
-				"skipUrlSync": false,
-				"type": "constant"
-			}
-		]
-	},
-	"time": {
-		"from": "now-15m",
-		"to": "now"
-	},
-	"timepicker": {},
-	"timezone": "browser",
-	"title": "Kepler Exporter Dashboard",
-	"uid": "NhnADUW4z",
-	"version": 2,
-	"weekStart": ""
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Dashboard for Kepler Exporter",
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 2,
+  "iteration": 1692621881727,
```

Comment:
I am not a grafana expert, so could you please help me understand why the `id` and the `iteration` has changed?